aur-sync: file inspection with PAGER="less -F" does not do the right thing

This is just a minor thing since using `less` for file inspection is not the best thing for me anyway, but still: if aur-sync decides to use `$PAGER` for file inspection which contains `less` using the `-F`/`--quit-if-one-screen` option, the build process does not pause before the inspection.

@lahwaacz I agree.

@AladW As `less` is part of the base group,`$PAGER` would end up never being used.

I think we could do something like this:

```diff
diff --git a/lib/aur-sync b/lib/aur-sync
index 7658e26..40ad76d 100755
--- a/lib/aur-sync
+++ b/lib/aur-sync
@@ -266,3 +266,3 @@ else
     # shellcheck disable=SC2086
-    viewer() ( cd "$1"; command -- ${PAGER:-less -K} )
+    viewer() ( cd "$1"; command -- ${PAGER:-less -K -+F} )
 fi
@@ -285,2 +285,6 @@ if ((view)); then
         } | viewer "$tmp_view"
+
+        if [[ -v PAGER ]]; then
+            read -p $'Press any key to continue or ctrl+d to abort\n'
+        fi
     fi
```
So the order becomes: `vifm` > `$PAGER` with aur prompt > `less -+F -K`

--- a/lib/aur-sync
+++ b/lib/aur-sync
@@ -264,7 +264,7 @@ if type -P >/dev/null vifm; then
     viewer() ( cd "$1"; vifm -c 'view!' -c '0' - )
 else
     # shellcheck disable=SC2086
-    viewer() ( cd "$1"; command -- ${PAGER:-less -K} )
+    viewer() ( cd "$1"; command -- ${PAGER:-less -K -+F} )
 fi
 
 # check if dependency graph is valid
@@ -283,6 +283,11 @@ if ((view)); then
           # print build directories in dependency order
           xargs -I{} -a "$tmp"/queue find -L "$tmp_view"/{} -maxdepth 1
         } | viewer "$tmp_view"
+
+        # Fix for #530
+        if [[ -v PAGER ]]; then
+            read -p $'Press any key to continue or ctrl+d to abort\n'
+        fi
     fi
 fi
 

